docs/combat-mechanics/turn-order/turn-order-mechanics.md

```diff
@@ -0,0 +1,111 @@
+---
+description: An explanation on how turn order is determined during combat.
+---
+
+# Turn Order Mechanics
+
+## Action Gauge
+
+Starts at 10000 by default for all units, once it reaches 0 for any given unit it is their turn to act. 
+
+## Gauge Action Modifier
+
+Various effects result in changes to a unit’s Action Gauge. 
+
```

Comment:
nit: Would be nice to have this also explain what happens in the event of ties after advancing (the advanced unit gets put at top of order in the case of ties)